cvat/apps/engine/views.py

```diff
@@ -49,7 +49,7 @@
 from cvat.apps.engine.frame_provider import FrameProvider
 from cvat.apps.engine.media_extractors import get_mime
 from cvat.apps.engine.models import (
-    ClientFile, Job, JobType, Label, SegmentType, Task, Project, Issue, Data,
+    ClientFile, Job, JobType, Label, AttributeSpec, SegmentType, Task, Project, Issue, Data,
```

Comment:
Remove unused import.

Reply:
> Remove unused import.

I have not imported this. This is here since 3 years ago.
![image](https://github.com/cvat-ai/cvat/assets/99531396/45e8d23c-1b3d-48f6-a290-612a14dbcc91)
Should i remove this??